configure.ac

```diff
@@ -4274,5 +4371,6 @@ AC_MSG_NOTICE([Configured to build curl/libcurl:
   metalink support: ${curl_mtlnk_msg}
   PSL support:      ${curl_psl_msg}
   HTTP2 support:    ${curl_h2_msg}
+  hsts support:     ${curl_hsts_msg} (using ${HSTS_FILE})
```

Comment:
Just a minor nit, lowercase `hsts` would look more readable (when referring to the standard) in uppercase `HSTS` above, [here](https://github.com/curl/curl/pull/2682/files#diff-67e997bcfdac55191033d57a16d1408aR3416), [here](https://github.com/curl/curl/pull/2682/files#diff-5b82be01b0af2106200b5ce04541e11bR2721) and [here](https://github.com/curl/curl/pull/2682/files#diff-8fb104c402dc51bdffef05a372f32aa2R1980).

Reply:
I agree with @vszakats!